content/spire/try/spire+aws-oidc-federation.md

```diff
@@ -117,11 +128,11 @@ spire-server   NodePort       10.12.13.51   <none>         8081:32115/TCP   10h
 
 At this point you can put the value of the spire-oidc EXTERNAL-IP in the required DNS A record, along with the value of the MY\_DISCOVERY\_DOMAIN domain that you specified in oidc-dp-configmap.yaml. For more details, see [Required DNS A Record for the OIDC Discovery Provider IP Address](#required-dns-a-record-for-the-oidc-discovery-provider-ip-address).
 
-## Create Client
+## Deploy the Client
 
-Apply the deployment file:
+Apply the client deployment file to create a client in a different namespace that you will use for testing in part 3 of this document:
```

Comment:
an internal link to `part 3` will be useful here

Reply:
Done.